### PR TITLE
coqPackages.relation-algebra: init

### PR DESCRIPTION
**Pull request by siraben**
###### Motivation for this change
Builds with Coq versions in Nixpkgs

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/coq-modules/aac-tactics/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  pname = "aac-tactics";
+
+  release."8.13.0".rev = "v8.13.0";
+  release."8.12.0".rev = "v8.12.0";
+  release."8.11.0".rev = "v8.11.0";
+  release."8.10.0".rev = "v8.10.0";
+  release."8.9.0".rev = "v8.9.0";
+  release."8.8.0".rev = "v8.8.0";
+  release."8.6.0".rev = "v8.6.0";
+  release."8.5.0".rev = "v8.5.0";
```
Comment (CohenCyril) [inline pkgs/development/coq-modules/aac-tactics/default.nix:14]: Please use `releaseRev = v: "v${v}";` instead

Comment (siraben) [reply pkgs/development/coq-modules/aac-tactics/default.nix]: Done.

File: pkgs/development/coq-modules/aac-tactics/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{ lib, mkCoqDerivation, coq, version ? null }:
+with lib;
+
+mkCoqDerivation {
+  pname = "aac-tactics";
+
+  releaseRev = v: "v${v}";
+
+  release."8.13.0".sha256 = "sha256-XdO8agoJmNXPv8Ho+KTlLCB4oRlQsb0w06aM9M16ZBU=";
```
Comment (vbgl) [inline pkgs/development/coq-modules/aac-tactics/default.nix:9]: > hash mismatch in fixed-output derivation '/nix/store/9ds4g7qk21xvdlc34f7k29qkiz4rahnd-source':
  wanted: sha256:05b4gb6z9356scqbvcah36hph81cwnjgis61pz7xb60919mbrlsx
  got:    sha256:0dk1lwcmb0rbrfr42fq59v7jfghhrb3lbsw94isrw2xkh9rwq29h

File: pkgs/development/coq-modules/aac-tactics/default.nix [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  inherit version;
+  defaultVersion = with versions; switch coq.coq-version [
+    { case = "8.13"; out = "8.13.0"; }
+    { case = "8.12"; out = "8.12.0"; }
+    { case = "8.11"; out = "8.11.0"; }
+    { case = "8.10"; out = "8.10.0"; }
+    { case = "8.9"; out = "8.9.0"; }
+    { case = "8.8"; out = "8.8.0"; }
+    { case = "8.6"; out = "8.6.0"; }
```
Comment (vbgl) [inline pkgs/development/coq-modules/aac-tactics/default.nix:26]: What about release `8.6.1`?

Comment (siraben) [reply pkgs/development/coq-modules/aac-tactics/default.nix:26]: I was not sure which version of Coq I could use with 8.6.1.

File: pkgs/development/coq-modules/relation-algebra/default.nix [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+mkCoqDerivation {
+  pname = "relation-algebra";
+  owner = "damien-pous";
+
+  releaseRev = v: "v${v}";
+
+  release."1.7.5".sha256 = "sha256-XdO8agoJmNXPv8Ho+KTlLCB4oRlQsb0w06aM9M16ZBU=";
+  release."1.7.4".sha256 = "sha256-o+v2CIAa2+9tJ/V8DneDTf4k31KMHycgMBLaQ+A4ufM=";
+  release."1.7.3".sha256 = "sha256-4feSNfi7h4Yhwn5L+9KP9K1S7HCPvsvaVWwoQSTFvos=";
+  release."1.7.2".sha256 = "sha256-B5TJayqiCBc86Qis0nNOv5KBcAGorqxjcCPRy8YmyMo=";
```
Comment (vbgl) [inline pkgs/development/coq-modules/relation-algebra/default.nix:13]: > hash mismatch in fixed-output derivation '/nix/store/i9p2qhbb7chs8v36wvzmd5ak9gg00fvb-source':
  wanted: sha256:1jn84v3cpl93f1isrbm805q834mz9rrx5b08x4y1f25259mwk507
  got:    sha256:0sh3z731rp1nyh11lkj0mm8xpayx1vk23q8bvhrw2d19gf6hv2kz

**Comment by vbgl** [comment]
@GrahamcOfBorg build coqPackages_8_10.relation-algebra

**Review by vbgl — CHANGES_REQUESTED** [review]
Please fix the hash for relation-algebra 1.7.2

File: pkgs/development/coq-modules/relation-algebra/default.nix (hunk 4 above)
Comment (siraben) [reply pkgs/development/coq-modules/relation-algebra/default.nix]: Fixed.

**Review by vbgl — APPROVED** [review]

**Pull request merged**